sys/posix/sockets/posix_sockets.c

```diff
@@ -1121,7 +1121,7 @@ int setsockopt(int socket, int level, int option_name, const void *option_value,
 #ifdef POSIX_SETSOCKOPT
     socket_t *s;
     struct timeval *tv;
-    const uint32_t max_timeout_secs = UINT32_MAX / (1000 * 1000);
+    const uint32_t max_timeout_secs = UINT32_MAX / (1000UL * 1000);
```

Comment:
```suggestion
    const uint32_t max_timeout_secs = UINT32_MAX / US_PER_SEC;
```
should make it clearer what's going on (from `time_units.h`)

Reply:
Updated as suggested.

Reply:
Did you push your changes?

Reply:
Not yet - thinking about the `time_t `definition issue for avr cpus.